src/Database/Expression/BetweenExpression.php

```diff
@@ -54,20 +54,20 @@ class BetweenExpression implements ExpressionInterface, FieldInterface
      * Constructor
      *
      * @param string|\Cake\Database\ExpressionInterface $field The field name to compare for values inbetween the range.
-     * @param mixed $from The initial value of the range.
-     * @param mixed $to The ending value in the comparison range.
+     * @param mixed $start The initial value of the range.
+     * @param mixed $end The ending value in the comparison range.
      * @param string|null $type The data type name to bind the values with.
      */
-    public function __construct($field, $from, $to, $type = null)
+    public function __construct($field, $start, $end, $type = null)
```

Comment:
I have never seen `start`, `end` used in SQL parlance for `BETWEEN`.

MySQL docs for e.g. uses "min", "max" https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#operator_between

Reply:
postgres just seems to call it a "range" with "endpoints". min and max makes sense to me, however, note that `BETWEEN 1 AND 3` is different than `BETWEEN 3 and 1` and I don't believe we sort anything.

Reply:
> is different than BETWEEN 3 and 1 and I don't believe we sort anything.

Yup, so we either keep the param names unchanged or if we want to be accurate name them `$value1`, `$value2` or `$expression1`, `$expression2`.